### ISSUE TITLE
It does not make sense to draw an image when [NSGraphicsContext currentContext] is nil

### ISSUE THREAD

**Issue by tresf** [LMMS/lmms#5582]
### Bug Summary

Master branch builds from CI won't load on macOS.  Below is the command line output:

```
mac:~ owner$ /Applications/LMMS.app/Contents/MacOS/lmms 
2020-07-15 16:48:51.241 lmms[44248:10366006] It does not make sense to draw an image when
    [NSGraphicsContext currentContext] is nil.  This is a programming error. Break on void
    _NSWarnForDrawingImageWithNoCurrentContext(void) to debug.  This will be logged only once.
    This may break in the future.
```

... apparently it did break in the future.  macOS 10.14 at least loads an empty GUI.  macOS 10.16 (beta) doesn't load any interface at all.

#### Affected LMMS versions

Master branch.  Tested download available in https://github.com/LMMS/lmms/pull/5579 on macOS 10.14, 10.16 (beta).

 * 10.14 - Loads a black screen, no usable UI components
 * 10.16 (beta) - Doesn't load anything at all, just hangs indefinitely

**Comment by PhysSong** [comment]
Test results:
- Qt 5.13 from Homebrew, for macOS 10.13: works fine on macOS 10.15
- Qt 5.15 from Homebrew, for macOS 10.13: broken on macOS 10.15
- Qt 5.15 from Homebrew, for macOS 10.15: works fine on macOS 10.15

Possibly related: https://bugreports.qt.io/browse/QTBUG-81370

**Comment by tresf** [comment]
@PhysSong thanks, so do you think this an upstream issue or is it something related to how we start the application or display the splash screen?  If it were that widespread, I'd expect people to be complaining in droves about this, yet they're not.

**Comment by PhysSong** [comment]
I'm not entirely sure. It might be from ABI incompatibility or something like that.

**Comment by tresf** [comment]
Quoting @PhysSong from #5584

> I've created macOS 10.14+ build with Qt 5.15.0 and tested it. It seems to fix both this and #5582. I will upload it to the release page and add a warning.

I've downloaded the 10.14+ build and can confirm that this works.
